myuw/templates/handlebars/card/bothell_application.html

```diff
@@ -0,0 +1,66 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "bothell_application_card" %}
+<div class="myuw-card application-card">
+{{#if is_returning}}
+    <h3 class="myuw-card-title">
+        Your Returning Student application
```

Comment:
Should "application" be capitalized here?

Reply:
yep